pkg/provisioner/resourcehandlers.go

```diff
@@ -131,6 +139,47 @@ func createObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryI
 	return
 }
 
+func updateObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryInterval, retryTimeout time.Duration) (result *v1alpha1.ObjectBucket, err error) {
```

Comment:
missing func comment

Reply:
Functions that are not exported in `Go` don't require comments, and it hasn't been the trend in this library from what I've seen to add comments to all functions regardless. I can add one here, but it is going against trend, and I don't think one is needed.